client/Dockerfile

```
@@ -10,19 +10,8 @@ RUN pnpm install --frozen-lockfile
 
 COPY . .
 
-ARG FACEBOOK_APP_ID
-ARG GOOGLE_ANALYTICS_ID
-ARG GOOGLE_API_KEY
-ARG GOOGLE_OAUTH_CLIENT_ID
-ARG SENTRY_INGEST_CLIENT
-ARG SENTRY_TRACE_RATE_CLIENT
-
-ENV VITE_APP_FACEBOOK_APP_ID=$FACEBOOK_APP_ID
-ENV VITE_APP_GOOGLE_ANALYTICS_ID=$GOOGLE_ANALYTICS_ID
-ENV VITE_APP_GOOGLE_API_KEY=$GOOGLE_API_KEY
-ENV VITE_APP_GOOGLE_OAUTH_CLIENT_ID=$GOOGLE_OAUTH_CLIENT_ID
-ENV VITE_APP_SENTRY_INGEST_CLIENT=$SENTRY_INGEST_CLIENT
-ENV VITE_APP_SENTRY_TRACE_RATE_CLIENT=$SENTRY_TRACE_RATE_CLIENT
+ARG CLARITY_PROJECT_ID
+ENV VITE_APP_CLARITY_PROJECT_ID=$CLARITY_PROJECT_ID
```

```suggestion
ENV VITE_CLARITY_PROJECT_ID=$CLARITY_PROJECT_ID
```

This didn't match the one in `client/src/index.tsx`, which used `VITE_CLARITY_PROJECT_ID`.
`APP` is not part of Vite requirements, so the name can be `VITE_APP_CLARITY_PROJECT_ID` too.